internal/driver/interactive_test.go

```diff
@@ -151,9 +162,10 @@ func newUI(t *testing.T, input []string) plugin.UI {
 }
 
 type testUI struct {
-	t     *testing.T
-	input []string
-	index int
+	t       *testing.T
+	input   []string
+	index   int
+	wantErr string
```

Comment:
A similar functionality exists in TestUI in proftest.go, but that lacks the ability to simulate the console input. Would it be better to extend that with the capability to simulate the input and reuse it here?

Reply:
Done.